src/ack_test_infra/resources.py

```diff
@@ -25,10 +25,14 @@
 
 from .aws import identity
 
-PLACEHOLDER_VALUES = {
-    "AWS_ACCOUNT_ID": identity.get_account_id(),
-    "AWS_REGION": identity.get_region(),
-}
+@property
+def PLACEHOLDER_VALUES():
```

Comment:
Any reason to make this into a property?

Reply:
The global values was (attempting to be) initialised when importing this script every time. It was causing errors when loading the package outside the scope of the PyTests.